creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -419,6 +419,25 @@ an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* there is an option to dissable bridge interface with pod network.
+This is done with a designated flag.
+The default value of the flag is 'true', meaning bridge interface is enabled.
+to disable it change flag value to false. For example:
+______________________________________________________________________________
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kube-system
+  labels:
+    kubevirt.io: ""
+data:
+ permitBridgeInterfaceOnPodNetwork: "false"
```

Comment:
looks like there is a single indentation space in this line but other lines use two spaces